src/genie/libs/parser/iosxe/c9500/show_platform.py

```diff
@@ -59,8 +59,12 @@ class ShowVersionSchema(MetaParser):
                     Optional('mem_size'): {
                         Any(): str,
                     },
-                    'license_level': str,
-                    'next_reload_license_level': str,
+                    Optional('license_level'): {
```

Comment:
I think we don't need to change these because both are corresponding to below.
```
        AIR License Level: AIR DNA Advantage

        Next reload AIR license Level: AIR DNA Advantage

Reply:
added new comment for this with suggested codes.